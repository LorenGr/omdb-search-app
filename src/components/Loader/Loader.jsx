import { Spin } from 'antd';

const Loader = () => {

    return <div className="spinner-container">
        <Spin size="large" />
    </div>;
}
export default Loader;