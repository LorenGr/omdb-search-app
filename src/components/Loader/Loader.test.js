// src/components/SearchComponent.test.js
import React from 'react';
import { render } from '@testing-library/react';
import Loader from './Loader';

it('renders and matches snapshot', () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
});
