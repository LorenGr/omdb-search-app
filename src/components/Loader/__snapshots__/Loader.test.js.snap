// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="spinner-container"
  >
    <div
      aria-busy="true"
      aria-live="polite"
      class="ant-spin ant-spin-lg ant-spin-spinning css-dev-only-do-not-override-dkbvqv"
    >
      <span
        class="ant-spin-dot ant-spin-dot-spin"
      >
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
      </span>
    </div>
  </div>
</DocumentFragment>
`;
